spec.html

```diff
@@ -1335,7 +1364,7 @@ <h1>Numeric Types</h1>
               BigInt::exponentiate
             </td>
             <td>
-              BigInt; may throw *RangeError*
+              a Completion Record normally containing a BigInt
```

Comment:
```suggestion
              a Completion Record (Abrupt, or Normal containing a BigInt)
```
what about this? the form also works for Return/Continue/Break

Reply:
If everyone else prefers this form I'll defer to that. Let's get a vote from @syg?

Reply:
I think we should just wait until we put all options on the table at the editor call.

Reply:
(It's a bit wordy, but on the other hand, you're clearly not going for conciseness, else you could say something like `Throw | Normal(BigInt)`.)